app/views/admin/organizations/new.html.erb

```diff
@@ -1,5 +1,5 @@
 <section class="content-header">
-<% content_for :title, "Organizations - New" %>
+<% content_for :title, "New - Organizations" %>
```

Comment:
My reading of the issue is that only the `current_organization.name` should be moved to the end of the string, not to flip all the titles.
@armahillo can you confirm?

Reply:
According the title of issue, I thought we have 2 actions:
+ Move action to first.
+ Move the "current_organization.name" to last.
so I implement both of them.

And the example of the issue:
actually: <% content_for :title, "#{current_organization.name} - Inventory - Barcode Items" %>
expected: <% content_for :title, "Barcode Items  - Inventory - #{current_organization.name}" %>
I see "Barcode Items" and "Inventory" also change position, not only move "current_organization.name" to the last. So in the code, I also changed it.

Should I revert it and move only "current_organization.name" to the end of the string.
Thanks!

Reply:
Nope, I think you had this right the first time around, and I misunderstood the issue.  This looks good - thanks @baonguyen1320!